elements/lisk-chain/test/unit/state_store/chain_state.spec.ts

```diff
@@ -25,9 +25,10 @@ describe('state store / chain_state', () => {
 	let stateStore: StateStore;
 	let db: any;
 
-	const lastBlockHeaders = ([{ height: 30 }, { height: 20 }] as unknown) as ReadonlyArray<
-		BlockHeader
-	>;
+	const lastBlockHeaders = ([
```

Comment:
why are the prettier changes getting affected? something changed? the earlier code was formatted too rite?

Reply:
I think prettier version changed the style?